WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -229,41 +229,46 @@
         </div>
       </div>
       <hr>
-      <%= f.input :registration_requirements, input_html: { class: "markdown-editor" } %>
-    <% end %>
-
-    <% disable_money_input = !@competition.can_edit_registration_fees? %>
-    <%= f.input :currency_code, collection: Money::Currency.table.values, label_method: lambda { |c| c[:name]  }, value_method: lambda { |c| c[:iso_code] }, disabled: disable_money_input  %>
-    <%= f.input :base_entry_fee_lowest_denomination, as: :money_amount, currency: @competition.currency_code, disabled: disable_money_input, currency_selector: "#competition_currency_code" %>
-
-    <%= f.input :use_wca_registration %>
+      <%= f.input :use_wca_registration %>
 
-    <div class="wca-registration-options">
-      <%= f.input :enable_donations %>
-
-      <%= f.input :guests_enabled, as: :radio_buttons, collection: [ :true, :false ] %>
+      <div class="wca-registration-options">
+        <% if @competition.can_receive_registration_emails?(current_user.id) %>
+          <%# Quick hack to fill receive_registration_emails with the correct value for the current_user %>
+          <% @competition.receive_registration_emails = @competition.receiving_registration_emails?(current_user.id) %>
+          <%= f.input :receive_registration_emails, as: :boolean %>
+        <% end %>
+        <%= f.input :enable_donations %>
+        <%= f.input :guests_enabled, as: :radio_buttons, collection: [ :true, :false ] %>
+      </div>
 
-      <% if @competition.can_receive_registration_emails?(current_user.id) %>
-        <%# Quick hack to fill receive_registration_emails with the correct
-            value for the current_user %>
-        <% @competition.receive_registration_emails = @competition.receiving_registration_emails?(current_user.id) %>
-        <%= f.input :receive_registration_emails, as: :boolean %>
-      <% end %>
+      <%= f.input :guests_entry_fee, as: :money_amount, currency: @competition.currency_code, currency_selector: "#competition_currency_code" %>
```

Comment:
I got rid of the `guests_entry_fee_type` as there are too many possibilities to cover. I think it's better to just treat this as the base entry fee, and any specification can go into the extra registration requirements field.

Reply:
Sounds good to me!